### ISSUE TITLE
SmartCity-Accessibility Server Side Network Tests #172 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#415]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.32 KB...]
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 7, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 44.621 sec <<< FAILURE! - in smartcity.accessibility.services.NavigationServiceTest
responseContainsLatLngArray(smartcity.accessibility.services.NavigationServiceTest)  Time elapsed: 38.867 sec  <<< FAILURE!
java.lang.AssertionError: Range for response status value 400 expected:<SUCCESSFUL> but was:<CLIENT_ERROR>
	at smartcity.accessibility.services.NavigationServiceTest.responseContainsLatLngArray(NavigationServiceTest.java:94)

Running smartcity.accessibility.navigation.NavigationTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 9.32 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 8, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.032 sec - in smartcity.accessibility.database.ParseDatabaseTest
Running smartcity.accessibility.search.NearbyPlacesTest
the length is : 10
lat is : 31.9028338 lng is : 34.9970901
lat is : 31.9075046 lng is : 34.9979821
lat is : 31.9075329 lng is : 34.997949
lat is : 31.9092985 lng is : 35.00175489999999
lat is : 31.9076557 lng is : 34.9981921
lat is : 31.908945 lng is : 34.996069
lat is : 31.9076557 lng is : 34.9981921
lat is : 31.9006521 lng is : 35.002661
lat is : 31.9005432 lng is : 35.0025467
lat is : 31.9121792 lng is : 35.00472180000001
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.101 sec - in smartcity.accessibility.search.NearbyPlacesTest

Results :

Failed tests: 
  NavigationServiceTest.responseContainsLatLngArray:94 Range for response status value 400 expected:<SUCCESSFUL> but was:<CLIENT_ERROR>

Tests run: 20, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:39 min
[INFO] Finished at: 2017-06-06T13:02:23+00:00
[INFO] Final Memory: 32M/158M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] df1b10ddea3b9cb6a1e1840e58c33f2f6b2ae044 - maven dependencies plugin #413

- [Alexander Kaplan] df2d8e8c35c0ae0f7991806dda4ff206e9ec35fc - procfile for heroku #413

- [Alexander Kaplan] 12bb806574e886ab57feb2dd084313977f4a012c - slashes are confusing #413

- [Alexander Kaplan] 14633708da692a7e4b1f874fdac5729623c4f376 - remove stdout print debug

- [Alexander Kaplan] 42a5263c78943ec214f4638a939f942b8ad9a6d0 - Java 8

- [Alexander Kaplan] aa55b8d6d835dd0bc4fee7b44f75413ae2b4b29b - modified gitignore

- [Alexander Kaplan] 1b3566e7b1badca249f54ea4564a1e89e1af7f77 - temp pom in root

- [Alexander Kaplan] fe7012684c51301944e913ec0a665b16d0bcca5f - edit temp pom #413

- [Alexander Kaplan] 8639659b2c858b01e3efca2237cc74e9efe70c13 - edit temp pom #413

- [Alexander Kaplan] 14f8880776ad80a4f344eb300e684b018ea63a3d - root change #413

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/172/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
